Gopkg.toml

```diff
@@ -80,8 +80,8 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/micrologger"
 
-[[constraint]]
-  branch = "master"
+[[override]]
+  branch = "metrics-cache-update"
```

Comment:
I don't think this was intended. Revert to master please.

Reply:
Already noticed. Fixing.